terraform/projects/app-ecs-instances/main.tf

```diff
@@ -216,3 +240,8 @@ output "available_azs" {
   value       = "${data.aws_subnet.subnets_for_prometheus.*.availability_zone}"
   description = "AZs available with running container instances"
 }
+
+output "asg_dev_scaledown_schedules" {
```

Comment:
I don't know if we particularly need to output this but fine either way

Reply:
Thought it would be useful to see what the schedule would be in case devs have a setting in their `tfvars` but have forgotten about it, as it won't be that visible unless the schedule is changed.